src/index.js

```diff
@@ -13,7 +13,11 @@
  */
 
 const preprocessEnvVar = require('./preprocessEnvVar');
```

Comment:
Can you remove these changes? We want to keep it as is so that the only permissible import is as documented in the readme.

Reply:
yes, updated this